data/modules/scripts/gamestore/gamestore.lua

```diff
@@ -6,25 +6,27 @@ OFFER_TYPE_ALLBLESSINGS
 ]]
 
 -- Parser
-dofile(CORE_DIRECTORY.. '/modules/scripts/gamestore/init.lua')
+dofile(CORE_DIRECTORY .. '/modules/scripts/gamestore/init.lua')
 -- Config
 
 HomeBanners = {
-	images = { "home/banner_riftwatcher.png" , "home/banner_runemaster.png" , "home/banner_podiumofrenown.png" },
+	images = { "home/banner_riftwatcher.png", "home/banner_runemaster.png", "home/banner_podiumofrenown.png" },
 	delay = 10
 }
 
+local offersNameComplement = (configManager.getBoolean(configKeys.VIP_SYSTEM_ENABLED) and "VIP") or "Premium Time"
+
 GameStore.Categories = {
+	-- Premium Time/VIP
 	{
-		--Premium Time
 		icons = { "Category_PremiumTime.png" },
-		name = "Premium Time",
+		name = (configManager.getBoolean(configKeys.VIP_SYSTEM_ENABLED) and "Vip Shop") or "Premium Time",
 		rookgaard = true,
-		state = GameStore.States.STATE_NONE,
+		state = (configManager.getBoolean(configKeys.VIP_SYSTEM_ENABLED) and GameStore.States.STATE_NEW) or GameStore.States.STATE_NONE,
```

Comment:
```suggestion
		state = GameStore.States.STATE_NONE,
```
change to state, This way it does not deduct the coins.